modules/src/ics04_channel/error.rs

```diff
@@ -129,6 +129,9 @@ pub enum Kind {
     #[error("Receiving chain block timestamp >= packet timeout timestamp")]
     LowPacketTimestamp,
 
+    #[error("Negativ chain timestamp: overflow ")]
+    NegativeChainTimestamp,
```

Comment:
I think calling this "chain timestamp" may give rise to confusion. Also, it would be nice to know what is the exact value that provoked this error.. the original `i64` (?) value that caused the error.

```suggestion
    #[error("Invalid timestamp in consensus state:{0}; timestamp must be a positive value")]
     NegativeChainTimestamp(int64),
```